src/soundio/soundmanager.cpp

```diff
@@ -148,24 +150,38 @@ QList<QString> SoundManager::getHostAPIList() const {
 void SoundManager::closeDevices(bool sleepAfterClosing) {
     //qDebug() << "SoundManager::closeDevices()";
 
+#ifdef __LINUX__
     bool closed = false;
+#else
+    Q_UNUSED(sleepAfterClosing);
+#endif
```

Comment:
This workaround does not feel like it belongs in a QML-specific PR.  If there is a problem with device opening/closing it should be separated out.  (We don't want QML work surprisingly changing things for the main branch)

Reply:
Ah yep, this is fixing [this issues](https://github.com/mixxxdj/mixxx/issues/14290), which freezes the UI for 5 second when you make an audio change on Linux. I can cherry pick this out in a separate PR or into a separate commit if that's okay with you?

Reply:
Fixed as 503004dd98dee3a1fa78bd6f6f9c104daac00242

There is some relation with the QML PR, so it would be great if we could keep it here and break it out.